cerberus/kubernetes/client.py

```diff
@@ -46,11 +46,28 @@ def monitor_nodes():
             logging.error("Exception when calling \
                            CoreV1Api->read_node_status: %s\n" % e)
         for condition in node_info.status.conditions:
-            if condition.type == "Ready":
-                node_status = condition.status
+            if condition.type == "ReadonlyFilesystem":
+                node_readonlyfilesystem_status = condition.status
+            elif condition.type == "KernelDeadlock":
+                node_kerneldeadlock_status = condition.status
+            elif condition.type == "MemoryPressure":
```

Comment:
Thinking about some of the conditions, kube will evict the pods from the node when it hits the condition meaning it will handle the condition to prevent the node from getting to unhealthy state. The condition status might flip during this period, debating if Cerberus should act on that event. My initial thought is to ignore them and just look at conditions which report a permanent problem. Thoughts?

Reply:
yes, maybe marking it failed for `*Pressure` is a bit extreme. We can leave them out. effectively just checking for `Ready` and `KernelDeadlock` conditions. What say?